calour/io.py

```diff
@@ -432,131 +516,105 @@ def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_m
 
     Returns
     -------
-    Experiment
+    MS1Experiment
 
     See Also
     --------
     read
     '''
-    logger.debug('Reading OpenMS data (OpenMS bucket table %s, map file %s)' % (data_file, sample_metadata_file))
-    if rows_are_samples:
-        data_file_type = 'openms_transpose'
+    if file_type == 'mzmine2':
+        if rows_are_samples is None:
+            rows_are_samples = False
+        if table_sep is None:
+            table_sep = ','
+        if direct_ids is None:
+            direct_ids = True
+        data_file_type = 'csv'
+    elif file_type == 'biom':
+        if rows_are_samples is None:
+            rows_are_samples = False
+        if direct_ids is None:
```

Comment:
again. for immutable variable, you don't need to set None as default.

Reply:
see above